src/utils/wallet.js

```diff
@@ -475,7 +475,15 @@ class Wallet {
     }
 
     async getLedgerAccountIds() {
-        const publicKey = await this.getLedgerPublicKey()
+        let publicKey
+        try {
+            publicKey = await this.getLedgerPublicKey()
+        } catch (error) {
+            if (error.id === 'U2FNotSupported') {
+                throw new WalletError(error.message, 'signInLedger.getLedgerAccountIds.U2FNotSupported')
```

Comment:
@marcinbodnar I'll merge as is, but it makes sense to convert into WalletError deeper in Ledger-related code, i.e. at least in `getLedgerPublicKey` or even more deeper

Reply:
@vgrichina Good point.